pkg/apis/controlplane/types.go

```diff
@@ -106,6 +106,19 @@ type ClusterGroupMembers struct {
 	CurrentPage       int64
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// GroupMembers is a list of GroupMember objects or IPBlocks that are currently selected by a Group.
+type GroupMembers struct {
```

Comment:
This struct is the same as ClusterGroupMembers. Do we need to redefine?

Reply:
And could not we reuse GroupMembers defined in `pkg/apis/controlplane/v1beta2/types.go`?

Reply:
I remember `controlplane/types.go` and `v1beta2/types.go` where intentionally kept the same, is this outdated?

Reply:
Humm.. I do not know this. Let us see what @tnqn would recommend.